README.md

```diff
@@ -1301,6 +1301,21 @@ Schemas can be used to parse values using `m/parse` and `m/parser`:
 ;               :children [[:primitive [:text "Hello, world of data"]]]}]]}]
 ```
 
+Parsing returns tagged values for `:orn`, `:catn`, `:altn` and `:multi`. For others it returns the value.
```

Comment:
The value is also not returned unchanged if there are e.g. nested `:orn`:s: `[:tuple [:orn [:foo ...` => `[(miu/-tagged :foo ...`

Reply:
good catch.